src/components/attendee/VideoComms/types.tsx

```diff
@@ -0,0 +1,94 @@
+import Twilio from "twilio-video";
+
+export enum VideoCommsStatus {
+  Disconnected = "DISCONNECTED",
+  Connecting = "CONNECTING",
+  Connected = "CONNECTED",
+  Errored = "ERRORED",
+}
+
+export interface VideoCommsProviderProps {
+  userId: string;
+  children: React.ReactNode;
```

Comment:
This should not be necessary, `React.FC<VideoCommsProviderProps>` will add the `children` field